src/PockyBot.NET/Services/Pegs/PegRequestValidator.cs

```diff
@@ -54,6 +54,9 @@ private void ValidateRecipient(Message message)
                 throw new PegValidationException(
                     $"You can't peg yourself.");
             }
+            if (message.MessageParts[2].UserId == _settings.BotId){
+                throw new PegValidationException("You can't give pegs to PockyBot.");
```

Comment:
Is this gonna be returned to the user? Do we want to word it more like "You can't give pegs to me?"

Reply:
Hmm good idea